src/app/pages/partners/results/result-grid.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
The functions didn't change much, but I extracted the Dialog to its own module, I moved the instance of the Dialog up a level to the Results, and of course added types. A side-by-side compare to the deleted result-grid.js would be useful.